packages/open-payments/package.json

```diff
@@ -32,16 +32,16 @@
   "devDependencies": {
     "@types/node": "^20.12.7",
     "@types/uuid": "^9.0.0",
-    "nock": "^13.3.0",
+    "nock": "14.0.0-beta.5",
```

Comment:
this is the [version](https://github.com/nock/nock) with experimental support for fetch